ChangeLog.txt

```diff
@@ -1,3 +1,7 @@
+Client library:
+- Add `mosquitto_delay_puback()` for delaying the PUBACK message
+  until the on_message callback returns.
+
```

Comment:
I think that is 

a) auto generated
b) added when adding a new release

Reply:
I don't think so https://github.com/eclipse/mosquitto/commit/2f8573b4562dde129860ff96ff794e173d99db11